internal/util/planparserv2/README.md

```diff
@@ -0,0 +1,23 @@
+# Generate Parser with Antlr4
+
+## Install Antlr4
+
+Please follow [install antlr4](https://github.com/antlr/antlr4/blob/master/doc/go-target.md) to install the antlr tool.
+
+The version of antlr tool: `4.10.1`.
```

Comment:
I can not run all unit test with 4.10.1 generated code.
4.9.3 works for me

Reply:
curl -O https://www.antlr.org/download/antlr-4.9.3-complete.jar

Reply:
My fault, I'll fix the doc error.